src/data/light.ts

```diff
@@ -56,7 +56,13 @@ export const lightSupportsDimming = (entity: LightEntity) => {
   );
 };
 
-export const getLightRgbColor = (entity: LightEntity): number[] | undefined =>
+export const getLightRgbColor = (
```

Comment:
Is this function still needed?

Reply:
Yeah, it is used for state badge

Reply:
I meant that since we use typescript, we can just inline it.